### PR TITLE
ref(settings-dialog) Refactor styles

### PR DESCRIPTION
**Pull request by robertpin**
Remove unused styles
Move some styles to their tab
Convert CalendarTab to TS
Fix mobile styles

### CODE REVIEW

File: react/features/settings/components/web/CalendarTab.tsx [hunk 1]
```diff
@@ -68,21 +73,41 @@ type State = {
...
+    };
 };
 
 /**
  * React {@code Component} for modifying calendar integration.
  *
  * @augments Component
  */
-class CalendarTab extends Component<Props, State> {
+class CalendarTab extends Component<IProps, State> {
```
Comment (horymury) [inline react/features/settings/components/web/CalendarTab.tsx:104]: add interface `IState` ?

File: react/features/settings/components/web/CalendarTab.tsx [hunk 2]
```diff
@@ -103,7 +128,7 @@ class CalendarTab extends Component<Props, State> {
      */
     componentDidMount() {
         this.props.dispatch(bootstrapCalendarIntegration())
-            .catch(err => logger.error('CalendarTab bootstrap failed', err))
+            .catch((err: Error) => logger.error('CalendarTab bootstrap failed', err))
```
Comment (horymury) [inline react/features/settings/components/web/CalendarTab.tsx:131]: nit: is this a  standar js Error?

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/settings/components/web/CalendarTab.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ interface IProps extends WithTranslation {
 /**
  * The type of the React {@code Component} state of {@link CalendarTab}.
  */
-type State = {
+type IState = {
```
Comment (horymury) [inline react/features/settings/components/web/CalendarTab.tsx:71]: interface? :)

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/settings/components/web/CalendarTab.tsx (hunk 3 above)
Comment (robertpin) [reply react/features/settings/components/web/CalendarTab.tsx]: done :)

**Review by horymury — APPROVED** [review]

**Comment by saghul** [comment]
I see we are using that 607px in a few places now. Can we please extract that to a constant to base/ui ?

**Pull request merged**